Bootstrap/SelectorTitleView/SelectorTitleView.swift

```diff
@@ -132,8 +135,16 @@ public class SelectorTitleView: UIView {
     // MARK: - Private
 
     private func updateArrowDirection() {
-        let asset: FinniversImageAsset = arrowDirection == .up ? .arrowUpSmall : .arrowDownSmall
-        button.setImage(UIImage(named: asset), for: .normal)
+        if #available(iOS 13.0, *) {
```

Comment:
Should we have a fallback for old iOS versions?

Reply:
My idea was to keep Boostrap injectable, not having any assets help us to use Swift Package Manager. Also, I think it's more flexible if you inject your own arrows but this class probably doesn't have the best API since it should be included in the `init` so it's clear that it needs that in order to be displayed properly.

Reply:
But we do inject images here, so what I mean is:
```swift
if #available(iOS 13.0, *) {
    ...
} else {
    button.setImage(arrowDirection == .up ? arrowUpImage : arrowDownImage, for: .normal)
}
```